webpack.config.babel.js

```diff
@@ -4,7 +4,7 @@ import webpack from 'webpack';
 
 export default {
     target: 'node',
-    entry: path.join(__dirname, 'lib/index.js'),
+    entry: ['babel-polyfill', path.join(__dirname, 'lib/index.js')],
```

Comment:
Yes, but in the StackOverflow's post, there is a comment which say that the require doesn't work in a specific case.